docs/_docs/snapshots/snapshots.adoc

```diff
@@ -178,9 +172,59 @@ the topology and wait while the data gets rebalanced and indexes are rebuilt.
 |N > M | Unsupported.
 |===
 
+=== Automatic snapshot restore procedure
+
+The automatic restore procedure allows the user to restore cache groups from a snapshot on an active cluster using the Java API or link:tools/control-script[command line script].
+
+Currently, this procedure has a number of limitations:
```

Comment:
Unless these limitations are getting removed in a known time frame, just list them:

This procedure has some limitations:

Otherwise, specify that they will be resolved.

Reply:
Thanks, I added ".. limitations, that will be resolved in future releases"